recipes/source.rb

```diff
@@ -61,6 +68,15 @@
   notifies :restart, 'service[prometheus]'
 end
 
+template node['prometheus']['flags']['config.file'] do
+  cookbook node['prometheus']['job_config_cookbook_name']
+  source node['prometheus']['job_config_template_name']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.